validator/testdata/feature_tests/ads.out

```diff
@@ -1042,17 +1042,7 @@ PASS
 |        data-siteid="37790885"
 |        data-wordads="1">
 |    </amp-ad>
-|
-|    <h2 class="broken">PulsePoint Header Bidding 300x250</h2>
-|    <amp-ad width="300" height="250"
-|        type="pulsepoint"
-|        data-pid="521732"
-|        data-tagid="76835"
-|        data-tagtype="hb"
-|        data-timeout="1000"
-|        data-slot="/1066621/ExchangeTech_Prebid_AdUnit">
-|    </amp-ad>
-|
+|  
```

Comment:
extra space? this is failing the integration test

Reply:
Fixed this, but travis build is still failing, logs dont seem to indicate what the problem is, and running `node build-system/pr-check.js` on my local machine also seems to be fine. Pls let me know if there is anything else to fix. Thanks!